apps/mull-ui/src/app/pages/profile/edit-profile/edit-profile.scss

```
@@ -0,0 +1,78 @@
+@import '../../../../variables.scss';
+
+.edit-profile-container,
+.edit-profile-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.edit-profile-container {
+  justify-content: center;
+  margin-bottom: 10px;
+
+  .mull-back-button {
+    align-self: flex-start;
+  }
+
+  .custom-file-upload-container {
+    position: relative;
+    margin: auto;
+    width: fit-content;
+    .custom-file-upload-image {
+      margin-left: auto;
+      margin-right: auto;
+      width: 10rem;
+      height: 10rem;
+      object-fit: cover;
+      border-radius: 50%;
+    }
+
+    .pencil-icon {
+      position: absolute;
+      top: 0.5rem;
+      right: 1rem;
+      border-radius: 20rem;
+      padding: 0.35rem;
+      border: 1px #27b09a solid;
+      color: #27b09a;
+      background-color: white;
+    }
+  }
+
+  .custom-text-input-container {
+    margin-bottom: 1.875rem;
+    width: 100%;
+  }
+
+  .description-label {
+    font-weight: 600;
+    line-height: 1.5;
+  }
+
+  .description-msg {
+    padding: 1em 2.5em 1em 1em;
+    outline: none;
+    resize: none;
+    border-radius: 20px;
+    border: 1.5px $primary-color solid;
+    &.error {
+      border-color: #ff0000;
+    }
+  }
+
+  .save-button,
+  .edit-header {
+    align-self: center;
+  }
+
+  .save-button {
+    margin-top: 1em;
+    width: 150px;
+  }
+
+  .edit-header {
+    font-size: 23px;
```

font-size should be in `rem`. This is nice because we can change the font-size of the `body` and it will apply everywhere! With `px` we'd have to go and change the font-size everywhere. I see we still have `px` in a few places, those should be cleaned up in a maintenance ticket.

`1.5rem` is `24px` in our app
```suggestion
    font-size: 1.5rem;
```

---

Done.